Ops handbook finance management (#833)
publiccodenet/about: activities/finance-management/reporting.md

---
type: Guide
explains: reporting requirements
---

# Reporting

> 

Reviewer: Suggestion: add an intro sentence for context.

```suggestion

We produce financial reports for different audiences throughout the year.
```
```

## Board meeting reports

A concise financial one-pager is produced by the Operations Coordinator for the bi-monthly board meeting. The one-pager gives an overview of:

* The current cash position
* Costs incurred
* Runway
* General information (depending on time of the year, e.g. updates on (new) funding sources, important budget changes and interim conclusion)

## Annual reporting

As an ANBI, we are legally obliged to publish two kinds of reports: an annual financial statement (Jaarrekening, see below) and an annual report (Jaarverslag). The latter is a powerful piece of communications that shows how we deliver on our mission, achieve the goals set out in our policy plan, and celebrate our accomplishments.

## Annual financial statement

> The Operations Coordinator is responsible for producing the annual financial statement (Jaarverslag), in close collaboration with our bookkeeper. The financial statement should include the following information:

Reviewer: Are the contents of the jaarverslag optional?
```suggestion
The Operations Coordinator is responsible for producing the annual financial statement (jaarverslag), in close collaboration with our bookkeeper. The financial statement must include the following information:
```

* The balance sheet
* The profit and loss statement
* An additional explanation

The financial statement must be published on our website within six months after the end of the financial year.

> [Template for our annual financial statement.](https://drive.google.com/drive/folders/1eeW-cCescwdGoxuCKwB4oMlhE5512Qu_)

Reviewer: ```suggestion
[Template for our annual financial statement](https://drive.google.com/drive/folders/1eeW-cCescwdGoxuCKwB4oMlhE5512Qu_).
```

## King Baudouin Foundation United States (KBFUS)

> We opened a fund at the KBFUS Foundation to enable smooth faciliation of receiving gifts from US donors. The name of our fund is [“Foundation for Public Code US Fund”.](https://kbfus.networkforgood.com/projects/52915-p-kbfus-funds-foundation-for-public-code-nl)

Reviewer: Is the word 'foundation' redundant in 'KBFUS Foundation'?

> We opened a fund at the KBFUS Foundation to enable smooth faciliation of receiving gifts from US donors. The name of our fund is [“Foundation for Public Code US Fund”.](https://kbfus.networkforgood.com/projects/52915-p-kbfus-funds-foundation-for-public-code-nl)

Reviewer: ```suggestion
We opened a fund at the KBFUS Foundation to facilite receiving gifts from US donors. The name of our fund is [“Foundation for Public Code US Fund”](https://kbfus.networkforgood.com/projects/52915-p-kbfus-funds-foundation-for-public-code-nl).
```

> The Operations Coordinator has log-in credentials to the grants interface where the fund is managed and reports can be filed. [More information about the fund and how the process works can be found on the Drive.](https://drive.google.com/drive/folders/1hv1XKbA6y8XYO9KqbKo5-TFlFxRROn0R)

Reviewer: ```suggestion
The Operations Coordinator has log-in credentials to the grants interface where the fund is managed and reports can be filed. [More information about the fund and how the process works](https://drive.google.com/drive/folders/1hv1XKbA6y8XYO9KqbKo5-TFlFxRROn0R) can be found on the Drive.
```

### KBFUS annual report

> The Foundation for Public Code needs to provide a narrative and financial report on the use of grant funds to KBFUS no less than annually and final reports upon completion of the grant period and until grant funds are expended in full. The reports should include a narrative account of how grant funds were used, what was accomplished by the expenditure of funds (including a description of progress made towards achieving the goals of the grant), and compliance with the terms of the Agreement, along with a financial statementand copies of any publications resulting from the grant.

Reviewer: ```suggestion
The Foundation for Public Code needs to provide a narrative and financial report on the use of grant funds to KBFUS no less than annually and final reports upon completion of the grant period and until grant funds are expended in full. The reports should include a narrative account of how grant funds were used, what was accomplished by the expenditure of funds (including a description of progress made towards achieving the goals of the grant), and compliance with the terms of the Agreement, along with a financial statement and copies of any publications resulting from the grant.
```

Reviewer: Does our agreement with them have any other description than 'Agreement'?

> The report should be signed by an authorized officer, and the financial statement should be attested by the responsible financial officer or a certified public accountant, and should reflect the categories stipulated in the attached approved budget. The report shall include a reconciliation of all amounts spent for the purpose specified, with the total grant proceeds received. [Detailed financial requirements can be found on our Drive.](https://drive.google.com/drive/folders/1Tv5j1vfn2MhDc2DtDmKCyQrrfkh2CJTj)

Reviewer: ```suggestion
The report should be signed by an authorized officer, and the financial statement should be attested by the responsible financial officer or a certified public accountant, and should reflect the categories stipulated in the attached approved budget. The report must include a reconciliation of all amounts spent for the purpose specified, with the total grant proceeds received. [Detailed financial requirements can be found on our Drive](https://drive.google.com/drive/folders/1Tv5j1vfn2MhDc2DtDmKCyQrrfkh2CJTj).
```

---
type: Guide
explains: reporting requirements
---

# Reporting

## Board meeting reports

A concise financial one-pager is produced by the Operations Coordinator for the bi-monthly board meeting. The one-pager gives an overview of:

* The current cash position
* Costs incurred
* Runway
* General information (depending on time of the year, e.g. updates on (new) funding sources, important budget changes and interim conclusion)

## Annual reporting

> Annual reporting consists of the annual financial statement and an annual report. The latter is a powerful piece of communications that shows how we deliver on our mission, and celebrate our accomplishments.

Reviewer 2: ```suggestion
As an ANBI, we are legally obliged to publish two kinds of reports: an annual financial statement (Jaarrekening, see below) and an annual report (Jaarverslag). The latter is a powerful piece of communications that shows how we deliver on our mission, achieve the goals set out in our policy plan, and celebrate our accomplishments.
```

> ## Financial annual statement

Reviewer 2: ```suggestion
## Annual financial statement
```

> Our first financial year runs from 1 July 2019 - 31 December 2020. The Operations Coordinator is working closely with the bookkeeper to produce the statement. An ANBI has to publish the financial statement on our website. The financial statement may concern the following information:

Reviewer 2: ```suggestion
The Operations Coordinator is responsible for producing the annual financial statement (Jaarverslag), in close collaboration with our bookkeeper. The financial statement should include the following information:
```

* The balance sheet
* The profit and loss statement
* An additional explanation

> The financial statement must be published on our website within 6 six months after the end of the financial year.

Reviewer 2: ```suggestion
The financial statement must be published on our website within six months after the end of the financial year.
```

[Template for our annual financial statement.](https://drive.google.com/drive/folders/1eeW-cCescwdGoxuCKwB4oMlhE5512Qu_)

## King Baudouin Foundation United States (KBFUS)

> The King Baudouin Foundation’s mission is to contribute to a better society. The Foundation is an actor for change and innovation, serving the public interest and increasing social cohesion in Belgium and Europe. They seek to maximize their impact by strengthening the capacity of organizations and individuals. They also stimulate effective philanthropy by individuals and corporations. The Foundation’s key values are integrity, transparency, pluralism, independence, respect for diversity, and promoting solidarity. The Foundation’s current areas of activity are social justice & poverty, health, heritage and culture, social engagement, Africa, Latin America, Asia, Education and development of talents, Europe, climate, the environment and biodiversity. The King Baudouin Foundation is a public benefit foundation.

Reviewer 2: ```suggestion
```

Reviewer 2: No need to include this information about them - this section is about reporting and what we need to deliver based on our legal agreements with them. Providing the link to the drive with all the background info should be sufficient.
